.gitlab/ci/sample_code.gitlab-ci.yml

```diff
@@ -271,12 +271,16 @@ getting_started_sample_code:
 
     ### https://docs.meltano.com/getting-started#run-a-data-integration-el-pipeline
 
-    - meltano schedule gitlab-to-postgres tap-gitlab target-postgres @daily
-
+    - meltano schedule add gitlab-to-postgres --extractor tap-gitlab --loader target-postgres --interval "@daily"
     - meltano schedule list
 
     - meltano add orchestrator airflow
 
+    ### quick job/run related sanity checks
```

Comment:
could definitely remove this, but I snuck in a quick sanity check to make sure scheduling jobs actually works.